crates/editor/src/hunk_diff.rs

```diff
@@ -139,31 +139,12 @@ impl Editor {
                             }
                         }
                     })
-                    .entry("Revert File", None, {
+                    .entry("Revert File", Some(RevertFile.boxed_clone()), {
```

Comment:
Would `.action(` work here? That would be a shorter declaration then.

https://github.com/zed-industries/zed/blob/207f1aea4772c3605f5f609dfce6b80375b63f18/crates/ui/src/components/context_menu.rs#L156

Reply:
Done